Kconfig

```diff
@@ -132,5 +132,15 @@ menu "Camera configuration"
         help
             Maximum value of DMA buffer
             Larger values may fail to allocate due to insufficient contiguous memory blocks, and smaller value may cause DMA interrupt to be too frequent
+    
+    config JPEG_FORMAT_EXTRA_BUFFER_SIZE
```

Comment:
I thank it's better to add a parameter to the `camera_config_t` struct rather than a menuconfig option.

Reply:
However, in most cases, I don't know what to set this value to be appropriate other than actual testing，do you?  
To avoid changing this value frequently in the driver，it's better to provide a way to change this value.